middleware/metrics.go

```diff
@@ -0,0 +1,14 @@
+package middleware
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// NewInvalidClusterValidations registers and returns a new counter metric server_request_invalid_cluster_validation_labels_total.
+func NewInvalidClusterValidations(reg prometheus.Registerer) *prometheus.CounterVec {
+	return promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+		Name: "server_request_invalid_cluster_validation_labels_total",
```

Comment:
```suggestion
		Name: "server_invalid_cluster_validation_label_requests_total",
```

☝️  We count requests, not labels.